docs/workflow/blocks/actions.rst

```diff
@@ -38,27 +46,109 @@ This example shows two buttons. The first one dispatches an asynchronous command
 block in order to start the inner workflow so that it runs and signals completion to the outer workflow.
 
 .. code-block:: json
+  {
+    "type": "actions",
+    "buttons": [
+      {
+        "label": "OK",
+        "color": "primary",
+        "blocks": [
+          {
+            "type": "dispatch",
+            "action": "resetApp"
+          }
+        ]
+      },
+      {
+        "label": "Cancel",
+        "blocks": [
+          {
+            "type": "init"
+          }
+        ]
+      }
+    ]
+  }
 
-                        {
-                          "type": "actions",
-                          "buttons": [
-                            {
-                              "label": "OK",
-                              "color": "primary",
-                              "blocks": [
-                                {
-                                  "type": "dispatch",
-                                  "action": "resetApp"
-                                }
-                              ]
-                            },
-                            {
-                              "label": "Cancel",
-                              "blocks": [
-                                {
-                                  "type": "init"
-                                }
-                              ]
-                            }
-                          ]
-                        }
+valueGetters
+++++++++++++++++++++++
+In this example, we are specifying which flow to load when the button is clicked. We do so using the `valueGetters` property and specifying an `adapter` and a `workflowId`.
+You can find this example on the `Flow Cloud <https://app.kendra.io/workflowCloud/listWorkflows>`, in the list of workflows.
+
+.. code-block:: json
+  {
+    "type": "actions",
+    "buttons": [
+        {
+            "label": "Launch",
+            "blocks": [
+                {
+                    "type": "launch",
+                    "valueGetters": {
+                        "adapter": "data.adapterName",
+                        "workflowId": "data.id"
+                    }
+                }
+            ],
+            "enabled": true // automatically added by the actions block. Not by a human.
```

Comment:
If comments are in some JSON, it is likely to cause an error when pasted. The comment could go in the explanation text instead. @gsambrotta

Reply:
Ahh, you are right! Hmm, I did add the explanation in the text, but I liked to have it highlighted in here too. 
Well, I guess we cannot.